projects/ngrx.io/content/guide/effects/index.md

```diff
@@ -18,79 +23,198 @@ npm install @ngrx/effects --save
+
+<div class="alert is-important">
+
+Event streams are not limited to dispatched actions, but can be _any_ observable that produces new actions, such as observables from the Angular Router, observables created from browser events, and other observable streams.
+
+</div>
+
+## Handling Errors
+
+Effects are built on top of observable streams provided by RxJS. Effects are listeners of observable streams that continue until an error or completion occurs. In order for effects to continue running in the event of an error in the observable, or completion of the observable stream, they must be nested within a "flattening" operator, such as `mergeMap`, `concatMap`, `exhaustMap` and other flattening operators. The example below shows the `loadMovies$` effect handling errors when fetching movies.
+
+<code-example header="movie.effects.ts">
 import { Injectable } from '@angular/core';
-import { HttpClient } from '@angular/common/http';
-import { Action } from '@ngrx/store';
 import { Actions, Effect, ofType } from '@ngrx/effects';
-import { Observable, of } from 'rxjs';
-import { catchError, map, mergeMap } from 'rxjs/operators';
+import { of } from 'rxjs';
+import { map, mergeMap } from 'rxjs/operators';
 
 @Injectable()
-export class AuthEffects {
-  // Listen for the 'LOGIN' action
+export class MovieEffects {
+
   @Effect()
-  login$: Observable&lt;Action&gt; = this.actions$.pipe(
-    ofType('LOGIN'),
-    mergeMap(action =>
-      this.http.post('/auth', action.payload).pipe(
-        // If successful, dispatch success action with result
-        map(data => ({ type: 'LOGIN_SUCCESS', payload: data })),
-        // If request fails, dispatch failed action
-        catchError(() => of({ type: 'LOGIN_FAILED' }))
+  loadMovies$ = this.actions$
+    .pipe(
+      ofType('[Movies Page] Load Movies'),
+      mergeMap(() => this.moviesService.getAll()
+        .pipe(
+          map(movies => ({ type: '[Movies API] Movies Loaded Success', payload: movies })),
+          catchError(() => of({ type: '[Movies API] Movies Loaded Error' }))
+        ))
       )
-    )
-  );
+    );
 
-  constructor(private http: HttpClient, private actions$: Actions) {}
+  constructor(
+    private actions$: Actions,
+    private moviesService: MoviesService
+  ) {}
 }
 </code-example>
 
-2.  Register the EffectsModule in your application root imports. This EffectsModule _must_ be added to
-    your root `NgModule` for the effects providers to be registered and start when your application is loaded.
+
+The `loadMovies$` effect returns a new observable in case an error occurs while fetching movies. The inner observable handles any errors or completions and returns a new observable so that the outer stream does not die. You still use the `catchError` operator to handle error events, but the empty observable returns an observable of a new action that is dispatched to the `Store`.
+
+## Registering root effects
```

Comment:
What about mentioning that you should always import `EffectsModule.forRoot`, even if you have no root effects to register?